themes/src/jupyter/base16-ocean-dark.css

```diff
@@ -1,71 +1,71 @@
-:matches(#notebook, body.api, body.blog) .highlight .py { color: #eff1f5 !important; } /* Name.Property */
-:matches(#notebook, body.api, body.blog) .highlight .nt { color: #96b5b4 !important; } /* Name.Tag */
-:matches(#notebook, body.api, body.blog) .highlight .nv { color: #bf616a !important; } /* Name.Variable */
-:matches(#notebook, body.api, body.blog) .highlight .ow { color: #96b5b4 !important; } /* Operator.Word */
-:matches(#notebook, body.api, body.blog) .highlight .w { color: #eff1f5 !important; } /* Text.Whitespace */
-:matches(#notebook, body.api, body.blog) .highlight .mf { color: #d08770 !important; } /* Literal.Number.Float */
-:matches(#notebook, body.api, body.blog) .highlight .mh { color: #d08770 !important; } /* Literal.Number.Hex */
-:matches(#notebook, body.api, body.blog) .highlight .mi { color: #d08770 !important; } /* Literal.Number.Integer */
-:matches(#notebook, body.api, body.blog) .highlight .mo { color: #d08770 !important; } /* Literal.Number.Oct */
-:matches(#notebook, body.api, body.blog) .highlight .sb { color: #a3be8c !important; } /* Literal.String.Backtick */
-:matches(#notebook, body.api, body.blog) .highlight .sc { color: #eff1f5 !important; } /* Literal.String.Char */
-:matches(#notebook, body.api, body.blog) .highlight .sd { color: #65737e !important; } /* Literal.String.Doc */
-:matches(#notebook, body.api, body.blog) .highlight .s2 { color: #a3be8c !important; } /* Literal.String.Double */
-:matches(#notebook, body.api, body.blog) .highlight .se { color: #d08770 !important; } /* Literal.String.Escape */
-:matches(#notebook, body.api, body.blog) .highlight .sh { color: #a3be8c !important; } /* Literal.String.Heredoc */
-:matches(#notebook, body.api, body.blog) .highlight .si { color: #d08770 !important; } /* Literal.String.Interpol */
-:matches(#notebook, body.api, body.blog) .highlight .sx { color: #a3be8c !important; } /* Literal.String.Other */
-:matches(#notebook, body.api, body.blog) .highlight .sr { color: #a3be8c !important; } /* Literal.String.Regex */
-:matches(#notebook, body.api, body.blog) .highlight .s1 { color: #a3be8c !important; } /* Literal.String.Single */
-:matches(#notebook, body.api, body.blog) .highlight .ss { color: #a3be8c !important; } /* Literal.String.Symbol */
-:matches(#notebook, body.api, body.blog) .highlight .bp { color: #eff1f5 !important; } /* Name.Builtin.Pseudo */
-:matches(#notebook, body.api, body.blog) .highlight .vc { color: #bf616a !important; } /* Name.Variable.Class */
-:matches(#notebook, body.api, body.blog) .highlight .vg { color: #bf616a !important; } /* Name.Variable.Global */
-:matches(#notebook, body.api, body.blog) .highlight .vi { color: #bf616a !important; } /* Name.Variable.Instance */
-:matches(#notebook, body.api, body.blog) .highlight .il { color: #d08770 !important; } /* Literal.Number.Integer.Long */
+#notebook .input_area, body.api pre, body.api code pre code,
+:matches(#notebook, body.api) .highlight,
```

Comment:
The `pre code` after `body.api code` needs to be removed in pretty much every style.

Reply:
OK will do.

Reply:
Done in https://github.com/StylishThemes/GitHub-Dark/pull/898/commits/2448cde7591d0f86dccb1f69c7463485043fb68d